projects/noloan/app/src/main/java/hasadna/noloan/SuggestionRecyclerAdapter.java

```diff
@@ -11,25 +11,29 @@
 import android.widget.Button;
 import android.widget.TextView;
 
-import hasadna.noloan.firestore.FirestoreClient;
 import hasadna.noloan.lawsuit.LawsuitActivity;
 import hasadna.noloan.protobuf.SmsProto.SmsMessage;
 import noloan.R;
 
 public class SuggestionRecyclerAdapter
     extends RecyclerView.Adapter<SuggestionRecyclerAdapter.RecyclerViewHolder> {
 
-  DbMessages suggestions;
+  DbMessages dbSuggestions;
 
   public SuggestionRecyclerAdapter() {
-    suggestions = DbMessages.getInstance();
+    dbSuggestions = DbMessages.getInstance();
```

Comment:
So it's all messages now. Maybe rename to `dbMessages`?

Reply:
:+1: